dev_tools/auto_merge/Dockerfile

```diff
@@ -16,23 +16,22 @@
 # This is a minimal Dockerfile to run the automerge script.
 #
 # Usage:
-#     docker build -t cirqbot -f dev_tools/automerge/Dockerfile .
+#     docker build -t cirqbot -f dev_tools/automerge/Dockerfile dev_tools
 #     docker run -it -e CIRQ_BOT_GITHUB_ACCESS_TOKEN=[access token] cirqbot
 #
 # Alternatively, if your container has access to the GCP Secret Manager, it can take the
 # value of the cirqbot-api-key secret.
 ########################################################################################
 
-FROM python:3.7-slim
+FROM python:3.7-slim@sha256:1689dea3fe5e2f230c3b2d0cc18b64e96f8214082c78792c67dbc1bda9b6c67c
```

Comment:
Why this?

Reply:
To achieve reproducible builds. It's basically pinning to a particular version.